CHANGELOG.md

```diff
@@ -1,5 +1,225 @@
 ### [Versions](https://material-ui.com/versions/)
 
+## 5.0.0-alpha.11
+
+###### _Sep 26, 2020_
+
+Big thanks to the 29 contributors who made this release possible.
+Here are some highlights ✨:
+
+- 👩‍🎨 A first iteration on the new styling solution.
+
+  You can find a [new version](https://next.material-ui.com/components/slider-styled/) of the slider in the lab powered by [emotion](https://emotion.sh/).
+
+  In the event you are already using styled-components in your application, you can swap emotion for styled-components 💅. Check [this CodeSandbox](https://codesandbox.io/s/sliderstyled-with-styled-components-2hjs0?file=/src/demo.tsx) for a demo, it relies on aliases to prevent any bundle size overhead.
+
+  The new styling solution saves 2kB gzipped in the bundle compared to JSS, and about 14 kB gzipped if you were already using emotion or styled-components.
+
+  Last but not least, the change allows us to leverate dynamic style props! We will leverate it for dynamic color props, variant props, and new style props (an improved [system](https://material-ui.com/system/basics/)).
+
+  This change has been in our roadmap for more than a year.
+  We have announced it in our [v4 release blog post](https://medium.com/material-ui/material-ui-v4-is-out-4b7587d1e701) as a direction v5 would take.
+- 🛠 A first iteration on the unstyled components.
+
+  You can find a [new version](https://next.material-ui.com/components/slider-styled/#UnstyledSlider.tsx) of the slider in the lab without any styles.
+  The unstyled component weights 6.5 kB gzipped instead of 26 kB for the styled version when used standalone. The component is best suited when you want to fully customize the look of the component without reimplementing the JavaScript and accessibility logic.
+
+- ⚡️ A first alpha of the [DataGrid](https://material-ui.com/components/data-grid/) component.
+
+  It took us 6 months of development since the initial commit (March 15th, 2020) to make the first alpha release of the grid. The component comes in two versions.
```

Comment:
```suggestion
  It has taken 6 months of development since the initial commit (March 15th, 2020) to make the first alpha release of the grid. The component comes in two versions:
```